versions/keycloak-client/26.0.1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "date": "2024-10-22",
+  "version": "26.0.1",
+  "blogTemplate": 3,
+  "documentationTemplate": 12,
+  "downloadTemplate": 24
```

Comment:
`keycloak-client` doesn't use `documentationTemplate` or `downloadTemplate` - those are specific to Keycloak core; so would remove these properties from here.

Reply:
Removed these fields from the template and the generated files.